### PR TITLE
Bug 1902584: note workaround for advanced networking issues

### PR DESCRIPTION
**Pull request by miabbott**

### CODE REVIEW

**Comment by miabbott** [comment]
I can create a new bug if needed, but this is changes based on feedback from the original BZ

**Comment by jboxman** [comment]
@codyhoag FYI

File: modules/installation-user-infra-machines-static-network.adoc [hunk 1]
```diff
@@ -81,4 +81,10 @@ enter the specific IP address you want and related information. For example:
 bond=bond0:em1,em2:mode=active-backup
 ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:bond0:none
 ----
+
+[IMPORTANT]
+====
+When using the advanced networking options, you may encounter issues during the first boot of {op-system} where the statically configured address is not present or not activated properly. In such cases, you may need to manually reboot the {op-system} machine in order to workaround this problem. In newer versions of {op-system}, the network code was significantly improved and this issue was resolved.  See link:https://bugzilla.redhat.com/show_bug.cgi?id=1902584[RHBZ#1902584] for additional details.
```
Comment (codyhoag) [inline modules/installation-user-infra-machines-static-network.adoc:87]:
```suggestion
When using the advanced networking options, you could encounter issues during the first boot of {op-system} where the statically configured address is not present or not activated properly. In such cases, you might need to manually reboot the {op-system} machine to workaround this problem. In newer versions of {op-system}, this issue is resolved. See link:https://bugzilla.redhat.com/show_bug.cgi?id=1902584[BZ#1902584] for additional details.
```

**Review by codyhoag — COMMENTED** [review]
@miabbott just some minor wordsmithing suggestions from me. Would this also apply to OCP 4.4? Or is this only relevant in 4.5?

**Comment by miabbott** [comment]
> @miabbott just some minor wordsmithing suggestions from me. Would this also apply to OCP 4.4? Or is this only relevant in 4.5?

Thanks for the suggestion!  This would be relevant to 4.4 as well

**Comment by codyhoag** [comment]
@miabbott looks great! If you can squash your commits, I'll go ahead and merge.

**Comment by miabbott** [comment]
Squashed + pushed :arrow_up:

**Pull request merged**